common/timeutil/time.go

```diff
@@ -188,3 +189,13 @@ func NsSinceBootTimeToTime(ns uint64) time.Time {
 func NsSinceEpochToTime(ns uint64) time.Time {
 	return time.Unix(0, int64(ns))
 }
+
+// roundToClosestN rounds a number to the closest multiple of n.
+func roundToClosestN(val int, n int) int {
+	return int(math.Round(float64(val)/float64(n))) * n
+}
```

Comment:
This is way specific for this but could lie in a math pkg by its own.

Reply:
It is only used here so I prefered to keep it here for now. If we see another package needs this, we can take it to a new math package